.ci_support/pyiron_config.py

```diff
@@ -0,0 +1,25 @@
+import os
```

Comment:
you moved the script.py to ./ci_support in a hidden directory. How is this supposed to work on my my local machine when developing the script? I would rather have a templates folder in the main directory that includes all the sub directories and the corresponding script.py and input.json. Is there anything that should be kept in mind doing so?
So I would move this pyiron_config into a folder under main resources, and there would be another folder templates that includes the scripts I intend to add.

Reply:
By the new commits, I tried to rearrange things out of the `.ci_support`. Now, only `pyiron_config` remains there. I will add a README later tonight which clarify how a similar local setup can be created.